packages/backend/src/config/chains.ts

```diff
@@ -0,0 +1,22 @@
+import { ChainId, UnixTime } from '@l2beat/shared-pure'
+
+const chainConfig = [
+  {
+    chainId: ChainId.ETHEREUM,
+    // Deployment of the first L2
+    minTimestamp: UnixTime.fromDate(new Date('2019-11-14T00:00:00Z')),
+  },
+  {
+    chainId: ChainId.ARBITRUM,
+    // ~ Timestamp of block number 0 on Arbitrum
+    minTimestamp: UnixTime.fromDate(new Date('2021-05-28T22:15:00Z')),
+  },
+]
+
+export function getMinTimestamp(chainId: ChainId): UnixTime {
```

Comment:
getChainMinTimestamp

Reply:
I mean chain is the argument so it speaks for itself, no?

Reply:
At one hand yes, but on the other I can very easily see the naming collision in the future which will make stating imports harder. I leave it up to you, I am not really pressing this change